reports/sb108/sb108.qmd

```diff
@@ -114,10 +113,23 @@ data <- ojo_tbl("case", .con = ojodb) |>
   ojo_collect()
 ```
 
-Result has **`r nrow(data) |> format(big.mark = ",")` rows**. Since only the 13 OSCN counties report the charges in each case, we can only look at those for this analysis of drug possession charges. **`r nrow(data_oscn) |> format(big.mark = ",")` of the rows** were from these counties, and **`r data_oscn |> filter(!is.na(count_as_filed)) |> nrow() |> format(big.mark = ",")` had non-missing values** for `count_as_filed`.
+Our initial data pull has **`r nrow(data) |> format(big.mark = ",")` rows**. Since only the 13 official OSCN counties report the charges in each case, we are limited to just those for this analysis of drug possession charges. **`r nrow(data_oscn) |> format(big.mark = ",")` of the rows** were from these counties, and **`r data_oscn |> filter(!is.na(count_as_filed)) |> nrow() |> format(big.mark = ",")` had non-missing values** for `count_as_filed`.
 
 * **Full list of OSCN counties:** `r paste(janitor::make_clean_names(oscn_county_list, "title"), collapse = ", ")`
 
+```{r}
+n_oscn_cases <- data |>
+  filter(district %in% oscn_county_list) |>
+  count(id) |>
+  nrow()
+
+n_total_cases <- data |>
+  count(id) |>
+  nrow()
+```
+
+In the data, there were `r format(n_oscn_cases, big.mark = ",")` unique **cases** filed in OSCN counties, compared to `r format(n_total_cases, big.mark = ",")` cases filed statewide. In other words, `r 100 * round(n_oscn_cases / n_total_cases, 4)`% of the cases were filed in the 13 OSCN counties. Accordingly, we can approximate what the statewide results might look like by assuming our end result accounts for roughly `r 100 * round(n_oscn_cases / n_total_cases, 2)`% of the total population affected by SB108.
```

Comment:
We could also do this by total # of charges or defendants instead of cases. Defendants actually might work better since that's what we're talking about here. Ryan I think also had just a constant multiplier that he would use (it was like 2.3 or something) to get an idea of how OSCN only filings would generalize to the rest of the state.